tests/playwright-tests/utils/generateUITheme.ts

```diff
@@ -0,0 +1,10 @@
+
+function generateUIThemeName(): string {
+    const date = new Date();
+    const utcFormat = date.toISOString().replace("T", " ").substring(0, 19);
+    const uithemename = 'UI Test'.concat(utcFormat);
```

Comment:
To avoid any warnings about unnecessary variable assignments, might just wanna change the last 2 lines to just be:

`return 'UI Test'.concat(utcFormat);`

Also I don't know about general TS conventions, but where we've done string concatenations elsewhere in the project, I think we've generally favoured using string literals like:

```return `UI Test${utcFormat}`;```